packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -118,44 +118,50 @@ export function makeScreenshotEditorComponent({ h, imageBuffer, dialog }: Factor
       setConfirmCrop(false);
       const handleMouseMove = makeHandleMouseMove(corner);
       const handleMouseUp = (): void => {
-        croppingRef.current && croppingRef.current.removeEventListener('mousemove', handleMouseMove);
+        DOCUMENT.removeEventListener('mousemove', handleMouseMove);
         DOCUMENT.removeEventListener('mouseup', handleMouseUp);
         setConfirmCrop(true);
       };
 
       DOCUMENT.addEventListener('mouseup', handleMouseUp);
-      croppingRef.current && croppingRef.current.addEventListener('mousemove', handleMouseMove);
+      DOCUMENT.addEventListener('mousemove', handleMouseMove);
     }
 
     const makeHandleMouseMove = useCallback((corner: string) => {
       return function (e: MouseEvent) {
+        if (!croppingRef.current) {
+          return;
+        }
+        const cropCanvas = croppingRef.current;
+        const mouseX = e.clientX - cropCanvas.getBoundingClientRect().x;
+        const mouseY = e.clientY - cropCanvas.getBoundingClientRect().y;
         switch (corner) {
           case 'topleft':
             setCroppingRect(prev => ({
               ...prev,
-              startx: Math.min(e.offsetX, prev.endx - 30),
-              starty: Math.min(e.offsetY, prev.endy - 30),
+              startx: Math.min(Math.max(0, mouseX), prev.endx - 33),
+              starty: Math.min(Math.max(0, mouseY), prev.endy - 33),
```

Comment:
nit, I would expect `startx` to be `startX` to be consistent with camelCase

(no need to change it in this PR though)